marshal/oracle/listpdb.go

```diff
@@ -18,27 +18,44 @@ package oracle
 import (
 	"bufio"
 	"bytes"
+	"github.com/ercole-io/ercole-agent/v2/marshal"
```

Comment:
Are you using a linter that split the imports?
Try https://golangci-lint.run/usage/linters/

Imports can't be mixed between standard libraries and other sources